apps/hubble/src/rpc/server.ts

```diff
@@ -981,16 +982,16 @@ export default class Server {
         const peer = Result.fromThrowable(() => call.getPeer())().unwrapOr("unknown");
         log.debug({ method: "getObjectsByFid", req: call.request }, `RPC call from ${peer}`);
 
-        const { fid, type, pageSize, pageToken, reverse } = call.request;
+        const { fid, type, tagOptions, pageSize, pageToken, reverse } = call.request;
 
-        const castsResult = await this.engine?.getObjectsByFid(fid, type, {
+        const castsResult = await this.engine?.getObjectsByFid(fid, type, tagOptions, {
           pageSize,
           pageToken,
           reverse,
         });
         castsResult?.match(
-          (page: MessagesPage<ObjectAddMessage>) => {
-            callback(null, messagesPageToResponse(page));
```

Comment:
interesting, so in getObjectsByFid you're directly constructing a result that looks like a Response type object thereby removing the need for `messagesPageToResponse` conversion